src/cotletico_manager.py

```diff
@@ -3,100 +3,16 @@
 import player
 import team
 import match
+import schedule
 
-
-def schedule_from_json(filename: str):
-    data = ''
-    with open(filename) as files:
-        text = files.read()
-        data = json.loads(text)
-
-    matches = list()
-    for match_info in data:
-        match = Match(match_info["Location"], 0, match_info["Date"], match_info["Team 1"], match_info["Team 2"], [], [])
-        matches.append(match)
-
-    schedule = Schedule(matches)
-    return schedule
-
-
-schedule = schedule_from_json('../data/schedule.json')
-
-
-def teams_from_json(filename: str):
-    data = ''
-    with open('../data/players.json') as files:
-        text = files.read()
-        data = json.loads(text)
-
-    with open('../data/flags.json') as file:
-        flags = json.load(file)
-
-    teams = list()
-    for team_name in data.keys():
-        players = list()
-        for player_info in data[team_name]:
-            pl = player.Player(**player_info)
-            players.append(pl)
-            pl.statistics = dict({'injured': False, 'red': False})
-        tm = team.Team(team_name, players, flag=flags[team_name])
-        teams.append(tm)
-    return teams
-
-
-teams = teams_from_json('../data/players.json')
+group_schedule = schedule.Schedule.schedule_from_json('../data/schedule.json')
```

Comment:
Please, rename this static method to _from_json_file_. 
It should turn out something like:
```
class Schedule:
...
@staticmethod
def from_json_file(filename: str):
	data = ''
    with open(filename) as files:
        text = files.read()
        data = json.loads(text)

    return from_json(data)
...
def from_json(data):
        matches = list()
        for match_info in data:
            m = match.Match(match_info["Match"], match_info["Location"], match_info["Date"], match_info["Team 1"], match_info["Team 2"], [], [])
            matches.append(m)

        schedule = Schedule(matches)
        return schedule
```